modules/ROOT/partials/deployment/services/webui_language_code.adoc

```diff
@@ -0,0 +1 @@
+IMPORTANT: For the time being, the embedded ownCloud Web frontent only supports the main language-code but does not handle any territory. When strings are available in the language-code `language_territory`, the web frontend does not see it as it only requests `language`. As impact, any translations made must exist in the requested `language` to avoid a fallback to the default.
```

Comment:
```suggestion
IMPORTANT: For the time being, the embedded ownCloud Web frontend only supports the main language code but does not handle any territory. When strings are available in the language code `language_territory`, the web frontend does not see it as it only requests `language`. In consequence, any translations made must exist in the requested `language` to avoid a fallback to the default.
```